dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
Automated build is a normal requirement for docker, but not so normal for development, where we often need to run a single phase, and not from scratch. So dobi would not replace fabric. OTH, fabric allows us to combine atomic tasks into a global task, so it can fulfill that specific dobi feature. What you say dobi gives us is better portability, and that is clearly valuable. However, is it portable because it always installs docker? We want to be able to run assembl "naked" on posix platforms, if possible.
I am curious about the steps of our fabfile install that fail on your arch setup; I never tried arch. If dobi can abstract those steps without docker, we're clearly interested.
Otherwise, as I told you earlier, I'm not against having dobi in the docker build chain; but I would personally want this chain to be optional.